tests/simulator/test_exceptions.py

```
@@ -0,0 +1,100 @@
+        super(BorisTwoCellJumpTest, self).__init__(*args, **kwargs)
+        self.simulator = None
+
+    def tearDown(self):
+        if self.simulator is not None:
+            self.simulator.reset()
+        self.simulator = None
+
+    def test(self):
+        simulation = ph.Simulation(**simArgs)
+        setup_model()
+        with self.assertRaises(Exception):
```

Ruff (0.13.1)

Do not assert blind exception: `Exception`

(B017)